js/shared_controls.js

```diff
@@ -1,3 +1,39 @@
+if (!Array.prototype.indexOf) {
```

Comment:
@Zarel is this what you wanted?

Reply:
Since both modes (1 vs All / All vs One / 1v1) use that method in their codes, I believed putting that polyfill at the very top of the file (which is the first file called after the datas) would be good to deny eventual problems in the future, although I'm pretty sure the browser will go through that condition wherever I would place it...